package.json

```diff
@@ -155,6 +156,7 @@
     "vite-plugin-eslint": "^1.8.1",
     "vite-tsconfig-paths": "4.2.0",
     "vitest": "0.32.2",
-    "webpack-bundle-analyzer": "^4.7.0"
+    "webpack-bundle-analyzer": "^4.7.0",
+    "xmlbuilder2": "^3.0.2"
```

Comment:
This needs to be added to the `dependencies` in `packages/platform/package.json` also